packages/core/src/internal/utils/identity.ts

```diff
@@ -16,6 +16,10 @@ export function isString(val: any): boolean {
   return is(String, val);
 }
 
+export function isNumericString(val: string): boolean {
```

Comment:
I added a utility function because I found myself repeatedly needing logic to test whether a string was a string representation of a number.